src/client/src/database.rs

```diff
@@ -86,6 +86,7 @@ impl Database {
                 catalog: self.catalog.clone(),
                 schema: self.schema.clone(),
                 authorization: self.ctx.auth_header.clone(),
+                dbname: format!("{}-{}", self.catalog, self.schema),
```

Comment:
@sunng87 please check this out whether it's desired

Reply:
I think I should update my patch first. Here if catalog and schema is provided, we can leave dbname a default empty string. I'm trying to  avoid leaking this concat pattern to client